docs/_index.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Cirq Developers. All Rights Reserved.
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
I think we should say "The Cirq Developers" as we do throughout cirq?

Reply:
Thanks for approval. I removed this license header since it's not necessary (should default to project license).
I'll check with the open source folks to clarify but I believe Google LLC is what they want.

Reply:
SGTM. Note that "Google LLC" is in the [AUTHORS file](https://github.com/quantumlib/Cirq/blob/master/AUTHORS) at the root of the repo.

Reply:
@lamberta please see go/opensource/releasing/authors.md for guidance - we chose the AUTHORS file method, which is kind of means The Cirq Authors / The Cirq Developers => list of authors. This allows for future extension of the authors without the need to update all the license headers.